### DISCUSSION TITLE
How to draw visualized picture of hemocells?

### DISCUSSION THREAD

**Discussion by Estufa-Ternura** [UvaCsl/HemoCell#22] [General discussion]
Hello,
What tool shall I use to generate pictures like FIGURE 8 in the paper <Cellular Level In-silico Modeling of Blood Rheology with An Improved Material Model for Red Blood Cells>?
Thanks!

**Comment by Estufa-Ternura** [comment]
the paper Cellular Level In-silico Modeling of Blood Rheology with An Improved Material Model for Red Blood Cells

**Reply by gzavo** [reply]
That figure was rendered using Paraview.
